maintainers/maintainer-list.nix

```diff
@@ -5518,6 +5518,16 @@
     githubId = 2396926;
     name = "Justin Woo";
   };
+  jvanbruegge = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
done